test/benchmarks/mdc/chips/app.module.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {MatChipsModule} from '@angular/material-experimental/mdc-chips';
+
+/** component: mdc-chip */
+
+@Component({
+  selector: 'app-root',
+  template: `
+    <button id="show-single" (click)="showSingle()">Show Single</button>
+    <button id="hide-single" (click)="hideSingle()">Hide Single</button>
+
+    <button id="show-multiple" (click)="showMultiple()">Show Multiple</button>
+    <button id="hide-multiple" (click)="hideMultiple()">Hide Multiple</button>
+
+    <mat-chip *ngIf="isSingleVisible">One</mat-chip>
+
+    <mat-chip-set *ngIf="isMultipleVisible">
```

Comment:
Aside from `mat-chip-set`, we also have a `mat-chip-grid` and `mat-chip-listbox` that should probably cover. Furthermore, there are other components like the chip avatar and remove button that aren't represented here.

Reply:
Done. I am a bit confused with `mat-chip-grid` and perhaps you could help me out here. If I do not provide an input within the `mat-chip-grid` alongside the `mat-chip-row`s the rows do not work properly. Am I doing something wrong or is this intentional?

[Here is a link to the html for this](https://github.com/angular/components/pull/20131/files#diff-26011fd879b21acc04111c8e2e884c78R25)